rosetta-source/src/main/rosetta/event-common-func.rosetta

```diff
@@ -1828,6 +1828,451 @@ func Qualify_Shaping: <"The qualification of a shaping event from the fact that
+                            value: newAssetQuantity,
+                            unit: unit,
+                                    ...
+                        }
+				then distinct
+
+			
+	alias newPriceQuantity: <"Create the new price quantity object.">
+        PriceQuantity {
+            price: [ newPrice ],
+            quantity: empty,
+            ...
+        }
+
+	set instruction -> split -> breakdown:
+		[ Create_TerminationInstruction( tradeState ) ]
+	
+
+
+        add instruction -> split -> breakdown: <"Sets the second part of the split to be a new contract with a new asset payout.">
+                    [
+                        PrimitiveInstruction {
+                            quantityChange: QuantityChangeInstruction {
+                                    change: [newPriceQuantity],
+                                    direction: QuantityChangeDirectionEnum -> Replace,
+                                    lotIdentifier: empty
+                                },
+                            termsChange: Create_RepriceOrAdjustmentTermsChangeInstruction(
+                                        tradeState -> trade -> tradableProduct -> product -> contractualProduct,
+                                        effectiveRepriceDate
+                                    ),
+                            ...
+                        }
+                    ]
+		
+		
+		
+func Qualify_Reprice: <"This qualification function is used to qualify repricing of a contractual product with an interest rate payout and assetPayout.">
+	[qualification BusinessEvent]
+	inputs:
+		businessEvent BusinessEvent (1..1)
+	output: is_event boolean (1..1)
+	
+	alias beforeTradePrice: businessEvent -> instruction only-element -> before -> trade -> tradableProduct -> tradeLot->priceQuantity
+	alias afterTradePrice: FilterOpenTradeStates( businessEvent -> after ) only-element -> trade -> tradableProduct -> tradeLot->priceQuantity
+	
+	alias beforeEconomicterms: businessEvent -> instruction only-element -> before -> trade -> tradableProduct -> product -> contractualProduct -> economicTerms
+	alias afterEconomicTerms: FilterOpenTradeStates( businessEvent -> after ) only-element -> trade -> tradableProduct -> product -> contractualProduct -> economicTerms
+
+	set is_event:
+		businessEvent -> after -> trade -> tradableProduct -> product -> contractualProduct -> economicTerms -> payout -> assetPayout exists and
+		businessEvent -> after -> trade -> tradableProduct -> product -> contractualProduct -> economicTerms -> payout -> interestRatePayout exists and
+		beforeTradePrice exists and
+		afterTradePrice exists and
+		afterTradePrice  <> beforeTradePrice and
+		beforeEconomicterms exists and
+		afterEconomicTerms exists and
+		afterEconomicTerms -> terminationDate = beforeEconomicterms -> terminationDate
+
+func Qualify_Adjustment: <"This qualification function is used to qualify repricing of a contractual product with an interest rate payout and assetPayout.">
```

Comment:
Same description as `Qualify_Reprice` - rogue copy/paste? Also the qualification logic looks identical...

Reply:
fixed

Reply:
Description for "Adjustment" still talks about "repricing".